deploy/srep-1000-ethtool-exporter/09-DaemonSet.yaml

```diff
@@ -0,0 +1,199 @@
+          # Taken from PROD 4.16.42 MCs currently using: @sha256:85765701a29f4a47596da38ecd9ec26a3a4173a952673c1bf8faf00ab2eb07cd
+          # Can also use image: prom/node-exporter
+          image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:85765701a29f4a47596da38ecd9ec26a3a4173a952673c1bf8faf00ab2eb07cd
+          command:
+            - /bin/sh
+            - -c
+            - |
+              export GOMAXPROCS=4
+              # We don't take CPU affinity into account as the container doesn't have integer CPU requests.
+              # In case of error, fallback to the default value.
+              NUM_CPUS=$(grep -c '^processor' "/proc/cpuinfo" 2>/dev/null || echo "0")
+              if [ "$NUM_CPUS" -lt "$GOMAXPROCS" ]; then
+                export GOMAXPROCS="$NUM_CPUS"
+              fi
+              echo "num_cpus=$NUM_CPUS gomaxprocs=$GOMAXPROCS"
+              exec /bin/node_exporter "$0" "$@"
+          env:
+            - name: DBUS_SYSTEM_BUS_ADDRESS
+              value: unix:path=/host/root/var/run/dbus/system_bus_socket
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 250m
+              memory: 180Mi
+            requests:
+              cpu: 8m
+              memory: 32Mi
+          securityContext: {}
+          volumeMounts:
+            - name: rootfs
+              mountPath: /host/root
+              readOnly: true
+            - name: sysfs
+              mountPath: /host/sys
+              readOnly: true
+            - mountPath: /var/node_exporter/text/file
+              name: node-exporter-textfile
+              readOnly: true
+            - name: procfs
+              mountPath: /host/proc
+              readOnly: true
+          workingDir: /var/node_exporter/textfile
+        - args:
+            - --secure-listen-address=[$(IP)]:9500
+            - --tls-cipher-suites=TLS_AES_128_GCM_SHA256,TLS_AES_256_GCM_SHA384,TLS_CHACHA20_POLY1305_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
+            - --upstream=http://127.0.0.1:9500/
+            - --tls-cert-file=/etc/tls/private/tls.crt
+            - --config-file=/etc/kube-rbac-policy/config.yaml
+            - --tls-min-version=VersionTLS12
+            - --tls-private-key-file=/etc/tls/private/tls.key
+            - --client-ca-file=/etc/tls/client/ca-bundle.crt
+          env:
+            - name: IP
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: status.podIP
+          # Taken from PROD 4.16.42 MCs currently using: sha256:82f5ad5a8c1827db890f6bbb9f85e3f571eceba5bc6903b721be6fdeaea651fa
+          image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:82f5ad5a8c1827db890f6bbb9f85e3f571eceba5bc6903b721be6fdeaea651fa
```

Comment:
I am not sure how much this version matters, but I presume it will get out of date pretty quickly. If the reality is this is all temporary, then maybe that is okay.

Reply:
Agree.  Depending on the timeline vs complexity, maybe can create some follow up tasks/ticket(s) to check and address this w/o automations for the near term?

Reply:
Can we use quay.io/prometheus/node-exporter:latest instead of the ocp image?